src/services/UserService.ts

```diff
@@ -112,11 +111,11 @@ export class UserService {
 			.getOne();
 
 		if (!user) {
-			throw new APIError(HttpCode.BadRequest, AuthenticateError.AccountNotFound);
+			throw new APIError(HttpCode.BadRequest, AuthenticateError.InvalidCredentials);
 		}
 
 		if (!verifyPassword(password, user.password)) {
-			throw new APIError(HttpCode.Forbidden, AuthenticateError.PasswordIncorrect);
+			throw new APIError(HttpCode.Forbidden, AuthenticateError.InvalidCredentials);
```

Comment:
Could we change all the error codes here to be `HttpCode.Forbidden`? Right now the difference in HTTP codes can be used to infer whether or not an email is registered

Reply:
consider it done